pages/services/beta-storage/0.1.0-beta/install/index.md

```diff
@@ -12,6 +12,11 @@ excerpt:
 - Storage feature flag is turned ON (`feature_dcos_storage_enabled: true` in config.yaml).
 - [DC/OS CLI](/latest/cli/install/) is installed and has been logged in as a superuser.
 
+# Download the package tarball
+
+DC/OS Storage Service (DSS) is packaged as a tarball.
+The tarball can be downloaded form [Mesosphere support site](https://support.mesosphere.com/).
```

Comment:
s/form/from

@jieyu I think is the link that we want to add:
https://support.mesosphere.com/hc/en-us/articles/213198586

You can reference the install instructions for Edge-LB:
https://docs.mesosphere.com/services/edge-lb/1.0.0/installing/